Source/Core/VideoCommon/VertexShaderGen.h

```diff
@@ -43,7 +43,7 @@ struct vertex_shader_uid_data
 
   u32 texMtxInfo_n_projection : 16;  // Stored separately to guarantee that the texMtxInfo struct is
                                      // 8 bits wide
-  u32 pad : 18;
+  u32 pad : 20;
```

Comment:
23+4+2+1+16 is 46, so a pad of 18 gives 64 bits total while a pad of 20 gives 66 bits total.  I'm not sure why this was changed; does anyone know?  It might just be a left-over from some other changes that previously adjusted the padding but no longer are relevant.

Reply:
Yeah, seems like it should be 18.